src/components/conversations/TemplateSelector.tsx

```
@@ -0,0 +1,381 @@
+              ))}
+            </>
+          )}
+        </div>
+
+        {/* Preview Footer */}
+        {hoveredTemplate && (
+          <div className="p-3 border-t bg-gray-50">
+            <div className="text-xs font-medium text-gray-600 mb-1">Preview:</div>
+            <div 
+              className="text-xs text-gray-700 line-clamp-3"
+              dangerouslySetInnerHTML={{ __html: getPreviewContent(hoveredTemplate) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)